atomicfu/build.gradle

```diff
@@ -156,7 +166,13 @@ task transformJS(type: JavaExec, dependsOn: [compileTestKotlinJs]) {
     outputs.file(transformedJsFile)
 }
 
-testMochaNode.dependsOn transformJS
```

Comment:
mocha tests are now disabled from `build` task?

Reply:
Custom js tests are disabled. Built-in tests 

* build
  * legacyjsTest 
    * legacyjsNodeTest
    * legacyjsBrowserTest
  * jsIrTest 
    * jsIrNodeTest
    * jsIrBrowserTest

are enabled